.travis.yml

```diff
@@ -11,6 +11,7 @@ env:
     - TEST=python NOSE_DIVIDED_WE_RUN=6a JS_SETUP=yes
     - TEST=python NOSE_DIVIDED_WE_RUN=bf JS_SETUP=yes
     - TEST=python-sharded-and-javascript JS_SETUP=yes
+    - TEST=python-elasticsearch-v2
```

Comment:
Hey @snopoke doesn't this setup the postgres DBs by default? The build for this matrix says the DB `test_commcarehq` doesn't exist.

Reply:
[Build](https://travis-ci.org/github/dimagi/commcare-hq/jobs/700026988)

```
======================================================================
ERROR: test suite for <module 'custom.icds_reports.tests.agg_tests' from '/mnt/commcare-hq-ro/custom/icds_reports/tests/agg_tests/__init__.py'>
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/vendor/lib/python3.6/site-packages/django/db/backends/base/base.py", line 213, in ensure_connection
    self.connect()
  File "/vendor/lib/python3.6/site-packages/django/db/backends/base/base.py", line 189, in connect
    self.connection = self.get_new_connection(conn_params)
  File "/vendor/lib/python3.6/site-packages/django/db/backends/postgresql/base.py", line 176, in get_new_connection
    connection = Database.connect(**conn_params)
  File "/vendor/lib/python3.6/site-packages/psycopg2/__init__.py", line 127, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: FATAL:  database "test_commcarehq" does not exist
```

Reply:
I think it's because this is being run before the DB setup. You need to find out why the `setUpModule`  (custom/icds_reports/tests/agg_tests/__init__.py", line 35, in setUpModule) is being run. Maybe try run tests with `--collect-only` and whatever other args you are using to see if it's finding any tests to run that would make it run that setup function